x-pack/plugins/index_lifecycle_management/server/routes/api/nodes/register_list_route.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+import { callWithRequestFactory } from '../../../lib/call_with_request_factory';
+import { isEsErrorFactory } from '../../../lib/is_es_error_factory';
+import { wrapEsError, wrapUnknownError } from '../../../lib/error_wrappers';
+import { licensePreRoutingFactory } from'../../../lib/license_pre_routing_factory';
+import { NODE_ATTRS_KEYS_TO_IGNORE } from './constants';
+
+function formatStats(stats) {
```

Comment:
Would it make sense to do this type of munging on the client? If so, then this code could just strip out disallowed node attributes, in which case a better name might be `removeBlacklistedStats`.

Reply:
I think it's better done on the server.  Renamed function to `convertStatsIntoList`.